apps/chrome-devtools/src/app-devtools/theming-panel/color.pipe.ts

```diff
@@ -1,16 +1,47 @@
 import { Pipe, type PipeTransform } from '@angular/core';
-import color from 'color';
+import TinyColor from 'tinycolor2';
+import { getAccessibilityContrastScore, getBestColorContrast } from './color.helpers';
 
+/**
+ * Convert the color to hexadecimal format
+ */
 @Pipe({
-  name: 'color',
+  name: 'hexColor',
   standalone: true
 })
-export class ColorPipe implements PipeTransform {
```

Comment:
isn't this a breaking change?

Reply:
That's an internal pipe expose only in this folder